src/datacollector/DataCollectorMain.cs

```diff
@@ -65,7 +70,15 @@ public void Run(string[] args)
             string logFile;
             if (argsDictionary.TryGetValue(LogFileArgument, out logFile))
             {
-                EqtTrace.InitializeVerboseTrace(logFile);
+                var traceLevelInt = CommandLineArgumentsHelper.GetIntArgFromDict(argsDictionary, TraceLevelArgument);
+
+                // In case traceLevelInt is not defined in PlatfromTraceLevel, default it to verbose.
+                var traceLevel = Enum.IsDefined(typeof(PlatformTraceLevel), traceLevelInt) ?
+                    (PlatformTraceLevel)traceLevelInt :
+                    PlatformTraceLevel.Verbose;
+
+                EqtTrace.Warning("DataCollectorMain.Run: Invalid trace level: {0}, defaulting to vebose tracelevel.", traceLevelInt);
```

Comment:
This will log always. Have a proper if for the previous block.
Nit:verbose*

Reply:
And without Initialize, will it work ?

Reply:
Corrected. thanks